### PR TITLE
Added Gale Token

### PR DESCRIPTION
**Pull request by niftybrain**

### CODE REVIEW

**Comment by niftybrain** [comment]
This is PR to get Gale listed on Defilama.

File: projects/gale/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const chain = config[ chainName ];
+    const totalDeposited = (await sdk.api.abi.call({
+      abi: windmillABI.totalBurned,
+      chain: chainName,
+      target: chain.windmillContract,
+      // params: [],
+      block: chainBlocks[ chainName ],
+    })).output;
+
+    const liquidityBalance = (await sdk.api.abi.call({
```
Comment (g1nt0ki) [inline projects/gale/index.js:41]: sorry, you cannot add entire LP as your tvl, if you have staking contract, you could add that value under pool2

File: projects/gale/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      abi: 'erc20:balanceOf',
+      chain: chainName,
+      target: chain.tokenContract,
+      params: chain.liquidityContract,
+      block: chainBlocks[ chainName ],
+    })).output;
+
+    const treasuryBalance = (await sdk.api.abi.call({
```
Comment (g1nt0ki) [inline projects/gale/index.js:49]: only non-native tokens can be added as part of 'treasury' section. Again, it cannot be marked as tvl.

Comment (g1nt0ki) [reply projects/gale/index.js:49]: ah, I see that you already have a separate treasury section, can you remove this code in tvl then, also update that section to only count non-native tokens

**Comment by llamatester** [comment]
The adapter at projects/gale exports TVL: 
        
 
 
    bsc                       1.51 M
    bsc-treasury              206.50 k
    treasury                  206.50 k
    
    total                    1.51 M

File: projects/gale/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+           transform =  await transformPolygonAddress();
+           break;
+      case "cronos":
+        transform =  await transformCronosAddress();
+        break;
+      default:
+        transform = await transformBscAddress();
+    };
+    const chain = config[ chainName ];
+    const totalDeposited = (await sdk.api.abi.call({
```
Comment (g1nt0ki) [inline projects/gale/index.js:33]: does this return the amount of GALE tokens that are burnt and never can be retrieved? if so, sorry but It cant be counted towards tvl, if the tokens can be retrieved by the user, it should still count towards 'staking' not tvl

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/gale exports TVL: 
        
 
 
    bsc                       0
    bsc-treasury              27.82 k
    bsc-staking               3.86 k
    treasury                  27.82 k
    staking                   3.86 k
    
    total                    0

**Pull request merged**